src/status_im2/contexts/wallet/create_account/backup_recovery_phrase/view.cljs

```diff
@@ -0,0 +1,93 @@
+(ns status-im2.contexts.wallet.create-account.backup-recovery-phrase.view
+  (:require
+    [quo.core :as quo]
+    [react-native.blur :as blur]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [status-im2.contexts.wallet.create-account.backup-recovery-phrase.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+;; Temporary words
+(def words ["witch" "collapse" "practice" "feed" "shame" "open" "lion"
+            "collapse" "umbrella" "fabric" "sadness" "obligue"])
+
+(defn word-item
+  [item index _ increment]
+  [rn/view {:style style/word-item}
+   [quo/counter {:type :grey} (+ index increment)]
+   [quo/text {:style {:margin-left 4}} item]])
+
+(defn words-column
+  [words first?]
```

Comment:
`first?` is a bit misleading here. I think you want `first-half?` or perhaps I misunderstood :)

Reply:
can we mark these helper components as private too?